docs/PYPI.md

```diff
@@ -74,61 +82,44 @@ following:
 
 </details>
 
-## Current Features
-
-- Discord Gateway communication
-- logging
-- Http Client
-- Events
-- Event middleware
-- Commands
-- Command arguments *(for types: str, int, float, bool, User, Channel, Role)*
-- Command argument choices
-- Command argument descriptions
-- Command cool downs (Using WindowSliding technique)
-- Tasks
-- Cogs
-
 **Client base class example:**
 
 ```py
 from pincer.client import Bot
 
 # Note that both `Bot` and `Client` are valid!
-bot = Bot("...")
+bot = Bot("YOUR_TOKEN_HERE")
 bot.run()
 ```
 
 **An example on the `on_ready` event**
 
+Pincer is designed to be used by inheriting the Client.
+
 ```py
 from time import perf_counter
-from pincer.client import Client
-
-client = Client("...")
-
-
-@client.event
-async def on_ready():
-    print(f"Logged in as {client.bot} after {perf_counter()} seconds")
+from pincer import Client
 
+class Bot(Client):
+    @Client.event
+    async def on_ready():
+        print(f"Logged in as {client.bot} after {perf_counter()} seconds")
 
+client = Bot("YOUR_TOKEN_HERE")
 client.run()
 ```
 
-### Inherited client
+### Interactions
 
-You have the possibility to use your own class to inherit from the Pincer bot
-base.
+Pincer is designed to make developing application commands intuitive and fast.
```

Comment:
Not a big fan of this *"Pincer is designed to [...]"* repetition, would refactor the sentence. Also both sentences don't sound the best in my opinion.

Reply:
> Pincer makes developing application commands intuitive and fast.
:thought_balloon:

Reply:
Im gonna go with that